packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -121,10 +127,26 @@ Spec.prototype.onException = function onException(e) {
     return;
   }
 
+  const {expected, actual} = e;
+  let message;
+  if (expected && actual) {
+    const diffString = diff(expected, actual, {
+      expand: this.expand,
+    });
+    message = matcherHint('.toBe') +
+      '\n\n' +
+      `Expected value to be (using ===):\n` +
+      `  ${printExpected(expected)}\n` +
+      `Received:\n` +
+      `  ${printReceived(actual)}` +
+      (diffString ? `\n\nDifference:\n\n${diffString}` : '');
+  }
+
   this.addExpectationResult(
     false,
     {
-      matcherName: '',
+      matcherName: 'blah blah',
```

Comment:
interesting. I think it's changing the way we deal with other types of errors too. Let me check out the branch and see what's going on